spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -899,6 +899,54 @@
       end
     end
 
+    describe "line 31 medical expenses" do
+      context "with a gross income of 200k" do
+        let(:submission) {
+          create :efile_submission, tax_return: nil, data_source: create(
+            :state_file_nj_intake,
+            :df_data_many_w2s,
+            medical_expenses: 567_890
+          )
+        }
+        it "writes sum $563,890.00 to fill boxes on line 31" do
```

Comment:
[dust] not to be a pedant but technically this is not a sum, it's a subtraction?

Reply:
Subtraction is just a sum with a negative number 😛